lib/server/server-middleware.js

```diff
@@ -45,6 +45,8 @@ const configureMiddleware = (options = {}) => {
     GA_TRACKING_ID
   } = options
 
+  const CSRF_TOKEN = SERVICE_TOKEN
```

Comment:
There's no need to re-assign the value of a variable just for the sake of the name when you're also destructuring:

```
const {
    serviceSources,
    ENV,
    PLATFORM_ENV,
    EDITABLE,
    SERVICE_SECRET,
    SERVICE_TOKEN: CSRF_TOKEN,
    FQD,
    USERNAME,
    PASSWORD,
    REALM,
    ASSET_PATH,
    ASSET_SRC_PATH,
    SENTRY_DSN,
    GA_TRACKING_ID
  } = options
```
This destructuring statement achieves the same thing

(And I'm not sure there's anything _gained_ by renaming the variable, anyway)

Reply:
Yes, you're right. I did that in order to sound clearer what it was used for, but you're alright. I am happy to change.

Reply:
Done! ♻️